### DISCUSSION TITLE
How does one specify a kubeconfig?

### DISCUSSION THREAD

**Discussion by avnermay** [argoproj-labs/hera#1248] [Q&A] [answered]
When using a yaml workflow, I can submit it to a remote server as follows without any issues:
```
argo submit -n argo --kubeconfig=/Users/avnermay/together/daringdolphin.kubeconfig hello-world.yaml
```

How do I get the equivalent behavior in Hera?

The following works for running on my local argo installation, but I'm not sure how to specify the kubeconfig for running this workflow remotely:

```
from hera.workflows import Steps, Workflow, WorkflowsService, script


@script()
def echo(message: str):
    print(message)


with Workflow(
    generate_name="hello-world-hera-",
    entrypoint="steps",
    namespace="argo",
    workflows_service=WorkflowsService(
        host="https://localhost:2746",
        verify_ssl=False,
    )
) as w:
    with Steps(name="steps"):
        echo(arguments={"message": "Hello world!"})

w.create()
```

**Comment by flaviuvadan** [comment, marked as answer]
Hey @avnermay! https://hera.readthedocs.io/en/stable/walk-through/authentication/#generating-a-k8s-token-via-tokengenerator might be helpful
